Pod/Classes/NotificationConvenience.swift

```diff
@@ -17,6 +17,11 @@ public extension NotificationCenter {
         }
     }
     
+    public func post(notificationName: Notification.Name, queue: DispatchQueue = DispatchQueue.main, object: Any? = nil, userInfo: [AnyHashable: Any]? = nil) -> Void {
```

Comment:
you should have your `public func post(notificationName: String` call this method